appveyor/test_task.bat

```diff
@@ -1,127 +1,129 @@
-if %errorlevel% neq 0 exit /b 3
-rem set OPENSSL_CONF=%OPENSSLDIR%\openssl.cnf
-set OPENSSL_CONF=
-rem set SSLEAY_CONF=
-
-rem prepare for OPcache
-if "%OPCACHE%" equ "1" set OPCACHE_OPTS=-d opcache.enable=1 -d opcache.enable_cli=1 -d opcache.protect_memory=1 -d opcache.jit_buffer_size=16M
-
-rem prepare for enchant
-mkdir C:\usr\local\lib\enchant-2
-if %errorlevel% neq 0 exit /b 3
-copy %DEPS_DIR%\bin\libenchant2_hunspell.dll C:\usr\local\lib\enchant-2
-if %errorlevel% neq 0 exit /b 3
-reg add HKEY_CURRENT_USER\SOFTWARE\Enchant\Config /v Module_Dir /t REG_SZ /d c:\enchant_plugins
-if %errorlevel% neq 0 exit /b 3
-set PHP_BUILD_CACHE_ENCHANT_DICT_DIR=%PHP_BUILD_CACHE_BASE_DIR%\enchant_dict
-if not exist "%PHP_BUILD_CACHE_ENCHANT_DICT_DIR%" (
-	echo Creating %PHP_BUILD_CACHE_ENCHANT_DICT_DIR%
-	mkdir "%PHP_BUILD_CACHE_ENCHANT_DICT_DIR%"
-)
-if not exist "%PHP_BUILD_CACHE_ENCHANT_DICT_DIR%\en_US.aff" (
-	echo Fetching enchant dicts
-	pushd %PHP_BUILD_CACHE_ENCHANT_DICT_DIR%
-	del /q *
-	powershell -Command wget http://windows.php.net/downloads/qa/appveyor/ext/enchant/dict.zip -OutFile dict.zip
-	unzip dict.zip
-	del /q dict.zip
-	popd
-)
-mkdir %LOCALAPPDATA%\enchant\hunspell
-copy %PHP_BUILD_CACHE_ENCHANT_DICT_DIR%\* %LOCALAPPDATA%\enchant\hunspell
-
-set PHP_BUILD_DIR=%PHP_BUILD_OBJ_DIR%\Release
-if "%THREAD_SAFE%" equ "1" set PHP_BUILD_DIR=%PHP_BUILD_DIR%_TS
-
-mkdir %PHP_BUILD_DIR%\test_file_cache
-rem generate php.ini
-echo extension_dir=%PHP_BUILD_DIR% > %PHP_BUILD_DIR%\php.ini
-echo opcache.file_cache=%PHP_BUILD_DIR%\test_file_cache >> %PHP_BUILD_DIR%\php.ini
-if "%OPCACHE%" equ "1" echo zend_extension=php_opcache.dll >> %PHP_BUILD_DIR%\php.ini
-rem work-around for some spawned PHP processes requiring OpenSSL
-echo extension=php_openssl.dll >> %PHP_BUILD_DIR%\php.ini
-
-rem remove ext dlls for which tests are not supported
-for %%i in (imap ldap oci8_12c pdo_firebird pdo_oci snmp) do (
-	del %PHP_BUILD_DIR%\php_%%i.dll
-)
-
-set TEST_PHPDBG_EXECUTABLE=%PHP_BUILD_DIR%\phpdbg.exe
-
-mkdir c:\tests_tmp
-
-set TEST_PHP_JUNIT=c:\junit.out.xml
-
-cd "%APPVEYOR_BUILD_FOLDER%"
-nmake test TESTS="%OPCACHE_OPTS% -q --offline --show-diff --show-slow 1000 --set-timeout 120 --temp-source c:\tests_tmp --temp-target c:\tests_tmp --bless %PARALLEL%"
-
-set EXIT_CODE=%errorlevel%
-
-appveyor PushArtifact %TEST_PHP_JUNIT%
-
-if %EXIT_CODE% GEQ 1 (
-	git checkout ext\pgsql\tests\config.inc
-	git diff > bless_tests.patch
-	appveyor PushArtifact bless_tests.patch
-)
-
-exit /b %EXIT_CODE%
+@echo on
+
+set NO_INTERACTION=1
+set REPORT_EXIT_STATUS=1
+set SKIP_IO_CAPTURE_TESTS=1
+
+if /i "%APPVEYOR_REPO_BRANCH:~0,4%" equ "php-" (
+	set BRANCH=%APPVEYOR_REPO_BRANCH:~4,3%
+	set STABILITY=stable
+) else (
+	set BRANCH=master
+	set STABILITY=staging
+)
+set DEPS_DIR=%PHP_BUILD_CACHE_BASE_DIR%\deps-%BRANCH%-%PHP_SDK_VS%-%PHP_SDK_ARCH%
+if not exist "%DEPS_DIR%" (
+	echo "%DEPS_DIR%" doesn't exist
+	exit /b 3
+)
+
+if "%APPVEYOR%" equ "True" (

```

Comment:
what is failing in this file?

for debug, I wrapped the DB services creation in never true condition, later, they should be created/provided using GH Action `services`